### ISSUE TITLE
Add user error 500

### ISSUE THREAD

**Issue by WeenderT** [dappur/framework#9]
```
Somewhere That Works

Error: Call to a member function get() on null in /var/www/html/new_app/app/src/Controller/AdminUsers.php:139 Stack trace: #0 [internal function]: Dappur\Controller\AdminUsers->usersAdd(Object(Slim\Http\Request), Object(Slim\Http\Response)) #1 /var/www/html/new_app/vendor/slim/slim/Slim/Handlers/Strategies/RequestResponseArgs.php(40): call_user_func_array(Array, Array) #2 /var/www/html/new_app/vendor/slim/slim/Slim/Route.php(344): Slim\Handlers\Strategies\RequestResponseArgs->__invoke(Array, Object(Slim\Http\Request), Object(Slim\Http\Response), Array) #3 /var/www/html/new_app/app/src/Middleware/Auth.php(12): Slim\Route->__invoke(Object(Slim\Http\Request), Object(Slim\Http\Response)) #4 [internal function]: Dappur\Middleware\Auth->__invoke(Object(Slim\Http\Request), Object(Slim\Http\Response), Object(Slim\Route)) #5 /var/www/html/new_app/vendor/slim/slim/Slim/DeferredCallable.php(43): call_user_func_array(Object(Dappur\Middleware\Auth), Array) #6 [internal function]: Slim\DeferredCallable->__invoke(Object(Slim\Http\Request), Object(Slim\Http\Response), Object(Slim\Route)) #7 /var/www/html/new_app/vendor/slim/slim/Slim/MiddlewareAwareTrait.php(73): call_user_func(Object(Slim\DeferredCallable), Object(Slim\Http\Request), Object(Slim\Http\Response), Object(Slim\Route)) #8 /var/www/html/new_app/app/src/Middleware/Admin.php(12): Slim\Route->Slim\{closure}(Object(Slim\Http\Request), Object(Slim\Http\Response)) #9 [internal function]: Dappur\Middleware\Admin->__invoke(Object(Slim\Http\Request), Object(Slim\Http\Response), Object(Closure)) #10 /var/www/html/new_app/vendor/slim/slim/Slim/DeferredCallable.php(43): call_user_func_array(Object(Dappur\Middleware\Admin), Array) #11 [internal function]: Slim\DeferredCallable->__invoke(Object(Slim\Http\Request), Object(Slim\Http\Response), Object(Closure)) #12 /var/www/html/new_app/vendor/slim/slim/Slim/MiddlewareAwareTrait.php(73): call_user_func(Object(Slim\DeferredCallable), Object(Slim\Http\Request), Object(Slim\Http\Response), Object(Closure)) #13 /var/www/html/new_app/vendor/slim/csrf/src/Guard.php(167): Slim\Route->Slim\{closure}(Object(Slim\Http\Request), Object(Slim\Http\Response)) #14 [internal function]: Slim\Csrf\Guard->__invoke(Object(Slim\Http\Request), Object(Slim\Http\Response), Object(Closure)) #15 /var/www/html/new_app/vendor/slim/slim/Slim/DeferredCallable.php(43): call_user_func_array(Object(Slim\Csrf\Guard), Array) #16 [internal function]: Slim\DeferredCallable->__invoke(Object(Slim\Http\Request), Object(Slim\Http\Response), Object(Closure)) #17 /var/www/html/new_app/vendor/slim/slim/Slim/MiddlewareAwareTrait.php(73): call_user_func(Object(Slim\DeferredCallable), Object(Slim\Http\Request), Object(Slim\Http\Response), Object(Closure)) #18 /var/www/html/new_app/vendor/slim/slim/Slim/MiddlewareAwareTrait.php(122): Slim\Route->Slim\{closure}(Object(Slim\Http\Request), Object(Slim\Http\Response)) #19 /var/www/html/new_app/vendor/slim/slim/Slim/Route.php(316): Slim\Route->callMiddlewareStack(Object(Slim\Http\Request), Object(Slim\Http\Response)) #20 /var/www/html/new_app/vendor/slim/slim/Slim/App.php(476): Slim\Route->run(Object(Slim\Http\Request), Object(Slim\Http\Response)) #21 /var/www/html/new_app/vendor/slim/slim/Slim/MiddlewareAwareTrait.php(122): Slim\App->__invoke(Object(Slim\Http\Request), Object(Slim\Http\Response)) #22 /var/www/html/new_app/vendor/slim/slim/Slim/App.php(370): Slim\App->callMiddlewareStack(Object(Slim\Http\Request), Object(Slim\Http\Response)) #23 /var/www/html/new_app/vendor/slim/slim/Slim/App.php(295): Slim\App->process(Object(Slim\Http\Request), Object(Slim\Http\Response)) #24 /var/www/html/new_app/public/index.php(30): Slim\App->run() #25 {main}
Call to a member function get() on null

Function: usersAdd
Class: Dappur\Controller\AdminUsers
Type: ->
Args: Array
File: /var/www/html/new_app/vendor/slim/slim/Slim/Handlers/Strategies/RequestResponseArgs.php
Line: 40
Function: call_user_func_array
Args: Array
File: /var/www/html/new_app/vendor/slim/slim/Slim/Route.php
Line: 344
Function: __invoke
Class: Slim\Handlers\Strategies\RequestResponseArgs
Type: ->
Args: Array
File: /var/www/html/new_app/app/src/Middleware/Auth.php
Line: 12
Function: __invoke
Class: Slim\Route
Type: ->
Args: Array
Function: __invoke
Class: Dappur\Middleware\Auth
Type: ->
Args: Array
File: /var/www/html/new_app/vendor/slim/slim/Slim/DeferredCallable.php
Line: 43
Function: call_user_func_array
Args: Array
Function: __invoke
Class: Slim\DeferredCallable
Type: ->
Args: Array
File: /var/www/html/new_app/vendor/slim/slim/Slim/MiddlewareAwareTrait.php
Line: 73
Function: call_user_func
Args: Array
File: /var/www/html/new_app/app/src/Middleware/Admin.php
Line: 12
Function: Slim\{closure}
Class: Slim\Route
Type: ->
Args: Array
Function: __invoke
Class: Dappur\Middleware\Admin
Type: ->
Args: Array
File: /var/www/html/new_app/vendor/slim/slim/Slim/DeferredCallable.php
Line: 43
Function: call_user_func_array
Args: Array
Function: __invoke
Class: Slim\DeferredCallable
Type: ->
Args: Array
File: /var/www/html/new_app/vendor/slim/slim/Slim/MiddlewareAwareTrait.php
Line: 73
Function: call_user_func
Args: Array
File: /var/www/html/new_app/vendor/slim/csrf/src/Guard.php
Line: 167
Function: Slim\{closure}
Class: Slim\Route
Type: ->
Args: Array
Function: __invoke
Class: Slim\Csrf\Guard
Type: ->
Args: Array
File: /var/www/html/new_app/vendor/slim/slim/Slim/DeferredCallable.php
Line: 43
Function: call_user_func_array
Args: Array
Function: __invoke
Class: Slim\DeferredCallable
Type: ->
Args: Array
File: /var/www/html/new_app/vendor/slim/slim/Slim/MiddlewareAwareTrait.php
Line: 73
Function: call_user_func
Args: Array
File: /var/www/html/new_app/vendor/slim/slim/Slim/MiddlewareAwareTrait.php
Line: 122
Function: Slim\{closure}
Class: Slim\Route
Type: ->
Args: Array
File: /var/www/html/new_app/vendor/slim/slim/Slim/Route.php
Line: 316
Function: callMiddlewareStack
Class: Slim\Route
Type: ->
Args: Array
File: /var/www/html/new_app/vendor/slim/slim/Slim/App.php
Line: 476
Function: run
Class: Slim\Route
Type: ->
Args: Array
File: /var/www/html/new_app/vendor/slim/slim/Slim/MiddlewareAwareTrait.php
Line: 122
Function: __invoke
Class: Slim\App
Type: ->
Args: Array
File: /var/www/html/new_app/vendor/slim/slim/Slim/App.php
Line: 370
Function: callMiddlewareStack
Class: Slim\App
Type: ->
Args: Array
File: /var/www/html/new_app/vendor/slim/slim/Slim/App.php
Line: 295
Function: process
Class: Slim\App
Type: ->
Args: Array
File: /var/www/html/new_app/public/index.php
Line: 30
Function: run
Class: Slim\App
Type: ->
Args: Array
```

first_name:Wender
last_name:Teixeira
email:weender641@gmail.com
username:teste
password:teste123
password_confirm:teste123
roles[]:auditor
perm_name[]:User
perm_value[]:true
perm_name[]:Developer
perm_value[]:true
dappurcsrf_name:dappurcsrf59b9fe3369aa6
dappurcsrf_value:1dc4a8b058d1320b89c6c6758381a7f2a4a7fe229f20f7929c9eeb17279ced7c
user_id:

**Comment by jeffmthomas** [comment]
I am planning on releasing a bug fix release that addresses this and a few other minor issues that I have noticed since the V2 release.

**Comment by jeffmthomas** [comment]
This was fixed in e44ba06.  It will be included in the v2.0.1 release tonight

**Comment by jeffmthomas** [comment]
This has been fixed with /releases/v2.0.1
